Documentation/HowToGuides/UsingCombinedActions/README.md

```diff
@@ -0,0 +1,205 @@
+
+![Import Combined Actions Prefab](assets/images/ImportCombinedActionsPrefab.png)
+
+Rename the newly created `Input.CombinedActions.BooleanTo1DAxisAction` GameObject to `Input.CombinedActions.BooleanTo1DAxisAction Horizontal`
+
+![RenameToHorizontal](assets/images/RenameToHorizontal.png)
+
+### Step 4
+
+In the Axis Settings on the `Boolean To 1DAxis Action` Facade of the `Boolean To 1DAxis Action Horizontal` GameObject, drag and drop the `Input.UnityInputManager.ButtonAction A` GameObject into the `Negative Input` parameter and drag and drop the `Input.UnityInputManager.ButtonAction D` GameObject into the `Positive Input` parameter.
+
+![Drag And Drop Into Horizontal Positive And Negative Parameters](assets/images/DragAndDropIntoHorizontalPositiveAndNegativeParameters.png)
+
+### Step 5
+
+Drag and drop another `Input.CombinedActions.BooleanTo1DAxisAction` prefab into the Unity hierarchy window. Rename the new `Input.CombinedActions.BooleanTo1DAxisAction` GameObject to `Input.CombinedActions.BooleanTo1DAxisAction Vertical`.
+
+![RenameToVertical](assets/images/RenameToVertical.png)
+
+In the Axis Settings on the `Boolean To 1DAxis Action` Facade of the `Boolean To 1DAxis Action Vertical` GameObject, drag and drop the `Input.UnityInputManager.ButtonAction S` GameObject into the `Negative Input` parameter and drag and drop the `Input.UnityInputManager.ButtonAction W` GameObject into the `Positive Input` parameter.
+
+![Drag And Drop Into Positive And Negative Parameters](assets/images/DragAndDropIntoVerticalPositiveAndNegativeParameters.png)
+
+### Step 6
+
+Create a new `Empty` Unity Object by selecting `Main Menu -> GameObject -> Create Empty` and rename it to `Movement`. Click `Add Component` and add a `Transform Position Mutator` component.
+
+![Add Position Mutator](assets/images/AddPositionMutator.png)
+
+### Step 7
+
+Drag and drop the `Capsule` GameObject into the `Target` parameter on the `Transform Position Mutator` Facade.
+
+![Drag And Drop Capsule](assets/images/DragAndDropCapsule.png)
+
+### Step 8
+
+Expand the `Tilia.Input.CombinedActions.Unity` Package directory in the Unity Project window and select the `Packages -> Tilia.Input.CombinedActions.Unity -> Runtime -> Prefabs` directory then drag and drop the `Input.CombinedActions.AxesToVector3Action` prefab into the Unity hierarchy window.
+
+![Import Axes To Vector3 Action](assets/images/ImportAxesToVector3Action.png)
+
+### Step 9
+
+On the `Input.CombinedActions.AxesToVector3Action` GameObject, Drag and drop the `Input.CombinedActions.BooleanTo1DAxisAction Horizontal` GameObject into the `Lateral Axis` property on the `Axes To Vector3` component.
+
+![Lateral Axis](assets/images/LateralAxis.png)
+
+and drag and drop the `Input.CombinedActions.BooleanTo1DAxisAction Vertical` GameObject into the `Longitudinal Axis` property on the `Axes To Vector3` component.
```

Comment:
Probably remove `and` from the start of the sentence and just start with `Drag and drop` (with an upper case d on Drag this time)

and again `Axes To Vector3 Action` component